docs/en/observability/observability-ui.asciidoc

```diff
@@ -1,33 +1,94 @@
-The dashboard is comprised of four components to help you make your environments 
-observable; APM, logs, metrics, and uptime. 
+[float]
+[[view-services-and-transactions]]
+== Services and transactions
 
-=== Services, transactions, and error rates
+The *APM* chart displays the total number of services running within your environment
+and the total number of transactions per minute that were captured by the Elastic APM
+agent instrumenting those services.
 
-coming[a future update]
+To drill-down and view this APM data in the *APM app*, click *View in app*.
 
-=== Log rates
+//TODO: what are the specific metric fields?
 
-coming[a future update]
+[role="screenshot"]
+image::images/apm.png[APM summary]
 
-=== System metrics
+[float]
+[[view-log-rates]]
+== Log rates
 
-coming[a future update]
+The *Logs* chart helps you to detect and inspect possible log anomalies across each of
+your ingested log sources. The visualization helps you determine if the log rate is outside
+of your expected bounds, and therefore could be considered anomalous. Any drop in the log
+rate could suggest a system has stopped responding, or a spike could denote a DDos attack.
 
-=== Systems availability 
+To drill-down and view these logs in the *Logs app*, click *View in app*.
 
-coming[a future update]
+//TODO: what are the specific metric fields?
 
-=== Alerts 
+[role="screenshot"]
+image::images/log-rate.png[Log rates]
 
-coming[a future update]
+[float]
+[[view-system-metrics]]
+== System metrics
+
+The *Metrics* chart provides you with a quick glance of the performance metrics
+aggregated on all of your running hosts. The metrics help you to evaluate the efficiency
+of hosts and determine whether there is any impact of resource consumption for your end-users.
+
+CPU usage, memory usage, and disk used metrics enable you to identify possible memory leaks,
+and whether disk resources need to be added or redistributed. Inbound and outbound traffic
+metrics provide you with a visualization of any possible spikes to help determine whether
+any hosts are choking your network. 
+
+To drill-down and view these performance metrics in the *Metrics app*, click *View in app*.
+
+//TODO: what are the specific metric fields?
+
+[role="screenshot"]
+image::images/metrics-summary.png[Metrics summary]
+
+[float]
+[[view-systems-availability]]
+== Systems availability 
+
+To help you react quickly to availability issues before they affect your users, the *Uptime*
+chart provides you with a snapshot of the overall availability of your hosts, network devices, or third-party
+services. Based on the last check reported by {heartbeat}, you can view the total number of detected monitors,
+along with the number of monitors in an `up` or `down` state.
+
+To drill-down and view this uptime data in the *Uptime app*, click *View in app*.
+
+//TODO: what are the specific metric fields?
+
+[role="screenshot"]
+image::images/uptime-summary.png[Uptime summary]
+
+[float]
+[[view-alerts]]
+== Alerts 
```

Comment:
This paragraph looks a bit off to me: alerting is not part of the introduction and as a user I only learn about it in the UI section. Any chance we can give alerting more visibility in this guide?

Reply:
I agree. I have added a section to the introduction.